src/inmanta/env.py

```diff
@@ -150,6 +151,19 @@ def activate_that(self) -> None:
         # Also set the python path environment variable for any subprocess
         os.environ["PYTHONPATH"] = os.pathsep.join(sys.path)
 
+        # write out a "stub" pip so that pip list works in the virtual env
+        bin_path = os.path.join(self.env_path, "bin", "pip")
```

Comment:
Shouldn't we call this `pip_path`?

Reply:
ok, change staged. But I want to wait for a build until everyone has their 2cents contributed :)